packages/runtime/container-runtime/src/opLifecycle/outbox.ts

```diff
@@ -180,10 +182,12 @@ export class Outbox {
 			this.params.containerContext.submitBatchFn === undefined
 		) {
 			// Nothing to do if the batch is empty or if compression is disabled or not supported, or if we don't need to compress
-			return batch;
+			return this.params.groupingManager.groupBatch(batch);
```

Comment:
It's not very clear (looking at changes to just that file) if it's correct. Why is grouping not involved on compression path?
Maybe add some comments?